### PR TITLE
UCP/PROTO: Fixed operation flags in proto v2 info.

### PR DESCRIPTION
**Pull request by rakhmets**
## What
Fixed printing operation flags when using UCX_PROTO_INFO.

## Why ?
Before:
image

After:
image

### CODE REVIEW

File: src/ucp/proto/proto_debug.c [hunk 1]
```diff
@@ -382,7 +385,7 @@ void ucp_proto_select_param_str(const ucp_proto_select_param_t *select_param,
         [ucs_ilog2(UCP_PROTO_SELECT_OP_FLAG_AM_RNDV)]  = "rndv"
     };
     unsigned op_flags                    = select_param->op_id_flags &
-                                           (UCP_PROTO_SELECT_OP_FLAGS_BASE - 1);
+                                           op_flags_bits;
```
Comment (yosefe) [inline src/ucp/proto/proto_debug.c:388]: maybe `& ~(UCP_PROTO_SELECT_OP_FLAGS_BASE - 1)` ?
or add helper function similar to ucp_proto_select_op_id - ucp_proto_select_op_flags

Comment (rakhmets) [reply src/ucp/proto/proto_debug.c]: Added.
+ updated code related to processing of op_flags.

File: src/ucp/proto/proto_debug.c [hunk 2]
```diff
@@ -384,15 +384,14 @@ void ucp_proto_select_param_str(const ucp_proto_select_param_t *select_param,
...
-                                           op_flags_bits;
-    uint32_t op_attr_mask;
+    uint32_t op_attr_mask, op_flags;
 
     ucs_string_buffer_appendf(
             strb, "%s", operation_names[ucp_proto_select_op_id(select_param)]);
 
     op_attr_mask = ucp_proto_select_op_attr_unpack(select_param->op_attr) &
                    op_attr_bits;
+    op_flags     = ucp_proto_select_op_flags(select_param) & op_flags_bits;
```
Comment (yosefe) [inline src/ucp/proto/proto_debug.c:394]: IMO we don't need the mask "op_flags_bits". all op flags could be printed.

File: src/ucp/proto/proto_select.inl [hunk 3]
```diff
@@ -79,6 +79,19 @@ ucp_proto_select_op_id(const ucp_proto_select_param_t *select_param)
...
+ucp_proto_select_check_op(const ucp_proto_select_param_t *select_param,
+                          uint64_t op_id_mask)
+{
+    return !!(UCS_BIT(ucp_proto_select_op_id(select_param)) & op_id_mask);
+}
+
+static UCS_F_ALWAYS_INLINE uint8_t
+ucp_proto_select_op_flags(const ucp_proto_select_param_t *select_param)
+{
+    return (select_param->op_id_flags & ~(UCP_PROTO_SELECT_OP_FLAGS_BASE - 1));
```
Comment (yosefe) [inline src/ucp/proto/proto_select.inl:92]: extra ( )

File: src/ucp/proto/proto_debug.c [hunk 4]
```diff
@@ -322,10 +322,10 @@ void ucp_proto_select_dump_short(const ucp_proto_select_short_t *select_short,
 static int
 ucp_proto_select_is_fetch_op(const ucp_proto_select_param_t *select_param)
 {
-    return ucp_proto_select_check_op(
-            select_param, UCS_BIT(UCP_OP_ID_GET) |
-                          UCS_BIT(UCP_OP_ID_RNDV_RECV) |
-                          UCS_BIT(UCP_OP_ID_AMO_FETCH));
+    return ucp_proto_select_check_op(select_param,
+                                     UCS_BIT(UCP_OP_ID_GET) |
```
Comment (yosefe) [inline src/ucp/proto/proto_debug.c:326]: IMO better align it on sam column (despite what clang-format is saying)

**Review by yosefe — APPROVED** [review]

**Review by yosefe — APPROVED** [review]

**Pull request merged**